scripts/setup.sh

```diff
@@ -2,12 +2,7 @@
 
 set -e
 
-export PATH=env/bin:${PATH}
```

Comment:
The nice thing about these scripts was that they could be run without needing to prefix them with any env magic. Is there a way to maintain that?